cmd/otel-agent/subcommands/run/command_test.go

```diff
@@ -21,8 +21,10 @@ func TestFxRun_WithDatadogExporter(t *testing.T) {
 	t.Setenv("DD_OTELCOLLECTOR_ENABLED", "true")
 	fxutil.TestRun(t, func() error {
 		ctx := context.Background()
-		params := &subcommands.GlobalParams{
-			ConfPaths: []string{"test_config.yaml"},
+		params := &cliParams{
```

Comment:
Don't have a super strong opinion, but it might be worth adding a test to check the pid file creation. Not sure if that test would land here or somewhere else, but it might be worth adding.

Reply:
Sounds good, added validation for this in the integration test